src/Validation/Validation.php

```diff
@@ -1216,10 +1282,10 @@ public static function uploadedFile($file, array $options = [])
         if ($options['optional'] && $error === UPLOAD_ERR_NO_FILE) {
             return true;
         }
-        if (isset($options['minSize']) && !static::fileSize($file, '>=', $options['minSize'])) {
+        if (isset($options['minSize']) && !static::fileSize($file, static::COMPARE_GREATER_THAN_OR_EQUAL_TO, $options['minSize'])) {
```

Comment:
What do you think about keeping them only as long as possible to be descriptive enough?
E.g. `COMPARE_GREATER_OR_EQUAL` would suffice IMO, same for others. The rest is gramm. sugar which shouldnt be needed for constants, otherwise they are long and make the rest of the code less readable.